waspc/README.md

```diff
@@ -39,16 +39,16 @@ the project. The best way to install it is via
 is a convenient way of installing and selecting versions of `cabal`, `hls` and
 `ghc`.
 
+:warning: If you are on a Mac, we still recommend you use the official [ghcup](https://www.haskell.org/ghcup/) installer over Homebrew, as the initial setup will be easier, and you won't have to worry about setting the correct environment variables.
+
 In [cabal.project](cabal.project) we're explicitly saying what compiler version
 to build Wasp with.
 
 ```cabal
 with-compiler: ghc-x.y.z
 ```
 
-Ensure that you have this [ghc
-version](https://downloads.haskell.org/ghc/latest/docs/html/users_guide/intro.html#ghc-version-numbering-policy)
-installed.
+Please ensure that you have this ghc version installed.
```

Comment:
Note: I removed the link here, because I found it confusing. I expected that it would take me to a page with the correct version of GHC, but it actually took me to their semantic version policy.

Reply:
Makes sense, I wonder why we put this link in the first place! Doesn't seem to be very helpful here.